### PR TITLE
add iscsi multipath and CHAP update

### PR DESCRIPTION
**Pull request by rootfs**
@childsb @ahardin-rh update doc to reflect latest iSCSI features in 3.6. Thanks

### CODE REVIEW

File: install_config/persistent_storage/persistent_storage_iscsi.adoc [hunk 1]
```diff
@@ -36,6 +36,8 @@ a volume in {product-title}. All that is required for iSCSI is iSCSI target
 portal, valid iSCSI IQN, valid LUN number, and filesystem type, and the
 `*PersistentVolume*` API.
 
+Optionally multipath portals and CHAP configuration can be provided.
```
Comment (ahardin-rh) [inline install_config/persistent_storage/persistent_storage_iscsi.adoc]: Optionally, multipath

Comment (ahardin-rh) [inline install_config/persistent_storage/persistent_storage_iscsi.adoc]: Challenge Handshake Authentication Protocol (CHAP) configuration

**Comment by rootfs** [comment]
@ahardin-rh feedback addressed, thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/persistent_storage/persistent_storage_iscsi.adoc#install-config-persistent-storage-persistent-storage-iscsi[Configuring Persistent Storage -> Persistent Storage Using iSCSI]
|Added information about multipath portals and Challenge Handshake Authentication Protocol (CHAP) configuration.
%

**Pull request merged**